doc/Caliptra.md

```diff
@@ -986,83 +988,82 @@ Each of Caliptra's internal anti-rollback fuse banks shall support a minimum cou
+Upon validating firmware version 4.4, the Caliptra RTM notes that that firmware's signed MIN\_SVN value is 2, and increments its internal fuse bank counter to match. After a chip receives firmware version 4.4, it is no longer able to roll back to version 4.2. However, it *will* continue to be able to roll back to version 4.3.
 
-## Physical Attack Countermeasures
+## Physical attack countermeasures
 
-A Caliptra RTM shall implement counter measures designed to deter both glitching (also referred to fault-injection (FI) and side-channel attacks (simple power analysis (SPA) and differential power analysis (DPA)).
+A Caliptra RTM shall implement countermeasures designed to deter both glitching (also referred to fault-injection (FI)) and side-channel attacks (simple power analysis (SPA) and differential power analysis (DPA)).
 
 The Caliptra threat model guides the priority of which physical countermeasures are based on a specific physical implementation.
 
-From the top, an adversary in the supply chain has essentially unlimited time to glitch the chip and make it reveal any private key material or symmetric secrets. [One Glitch To Rule Them All ](https://arxiv.org/abs/2108.04575)is one example with recency bias. The most critical counter-measures must prevent non-destructive extraction of those secrets. Otherwise, an adversary succeeding may be able to silently impersonate production serving assets at a later time.
+From the top, an adversary in the supply chain has essentially unlimited time to glitch the chip and make it reveal any private key material or symmetric secrets. [One Glitch To Rule Them All ](https://arxiv.org/abs/2108.04575)is one example with recency bias. The most critical countermeasures must prevent non-destructive extraction of those secrets. Otherwise, an adversary who succeeds can silently impersonate production-serving assets at a later time.
 
-General protection of the embedded microprocessor while running arbitrary firmware is required to protect the UDS or other private entropy from logical and physical attacks, including firmware running within the Caliptra itself. Control flow integrity, analog reference voltage and clock sources, as well as defensive programming are encouraged.  Likewise, pointer authentication, encryption, separate code vs data stacks, and memory tagging are all encouraged.
+General protection of the embedded microprocessor while running arbitrary firmware is required to protect the UDS or other private entropy from logical and physical attacks, including firmware running within Caliptra itself. Control flow integrity, analog reference voltage, and clock sources, as well as defensive programming, are encouraged.  Likewise, pointer authentication, encryption, separate code vs. data stacks, and memory tagging are all encouraged.
```

Comment:
This specification shouldn't be "encouraging" implementers. In addition the "encouragement requirements" are vague. What does "clock sources" even mean? I'd say we can remove this paragraph for now.

Reply:
removed